x-pack/plugins/security_solution/public/common/components/hover_actions/utils.ts

```diff
@@ -18,3 +20,20 @@ export const getAdditionalScreenReaderOnlyContext = ({
 
   return Array.isArray(value) ? `${field} ${value.join(' ')}` : `${field} ${value}`;
 };
+
```

Comment:
Extract duplicated logic and make it a hook for reusability

Reply:
can you please add a little unit test here?